src/server/req_rerun.c

```diff
@@ -362,7 +362,7 @@ timeout_rerun_request(struct work_task *pwt)
 		conn_idx = connection_find_actual_index(pjob->ji_rerun_preq->rq_conn);
 	}
 	reply_text(pjob->ji_rerun_preq, PBSE_INTERNAL,
-		"rerunning job has timed out");
+		"rerunning job is still in progress for");
```

Comment:
I don't see the change that @arungrover requested regarding returning zero. I believe a more accurate message might be:
Response timed out. Job rerun request still in progress.